packages/ui/src/memberships/modals/BuyMembershipModal/BuyMembershipModal.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import { Meta, Story } from '@storybook/react'
+import React from 'react'
+import { MemoryRouter } from 'react-router-dom'
+
+import { BuyMembershipModal } from '@/memberships/modals/BuyMembershipModal/BuyMembershipModal'
+import { MockApolloProvider } from '@/mocks/components/storybook/MockApolloProvider'
+
+export default {
+  title: 'Member/Modals/BuyMembershipModal',
```

Comment:
Nice :raised_hands: 

I had not seen that you added this one when I asked for the `SocialMediaSelector` story. But having `SocialMediaSelector` is still good, it will be useful for adding some tests in the tests refactor.

Reply:
Yeah, story might have been added after your review. It's present under `packages/ui/src/memberships/components/SocialMediaSelector/SocialMediaSelector.stories.tsx`